tests/foreman/conftest.py

```diff
@@ -114,18 +115,16 @@ def log_test_execution(robottelo_logger, test_name):
     robottelo_logger.debug(f'Finished Test: {test_name}')
 
 
-@pytest.fixture
-def test_name(request):
```

Comment:
This was unused, and not helpful, as its a one line call that can be made by any test or fixture requiring it.

Reply:
not quite unused... making some updates to where this was used. Initial search didn't find anything... thanks pycharm.